jira/client.py

```diff
@@ -2524,6 +2524,9 @@ def create_issue_link(
         issue_link_types = self.issue_link_types()
 
         if type not in issue_link_types:
+            LOG.warning(
```

Comment:
```suggestion
            self.log.warning(
```

We use this logger object !